packages/terra-form-field/src/Field.jsx

```diff
@@ -131,6 +133,48 @@ const Field = (props, { intl }) => {
     labelAttrs.className,
   ]);
 
+  // Checks to run when not in production
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
I take it we will remove this in the next major version of this component. Should we leave a comment noting to remove this section at a later date?

Reply:
Yeah thats a good idea, I'll log an issue as well to reference in the code comment.

Reply:
Update in this commit: https://github.com/cerner/terra-core/pull/2273/commits/10798ad56d452ec1cec05272bbe5314cefc07517
Logged to follow up: https://github.com/cerner/terra-core/issues/2290